sdks/python/apache_beam/testing/load_tests/build.gradle

```diff
@@ -34,8 +34,8 @@ task run(type: Exec, dependsOn: installGcpTest) {
 
     def runnerArg = project.findProperty(runnerProperty) ?: ""
     if (runnerArg == 'DataflowRunner' || runnerArg == 'TestDataflowRunner') {
-        dependsOn 'sdist'
-        loadTestArgs +=" --sdk_location=${project.buildDir}/apache-beam.tar.gz"
+        dependsOn ':sdks:python:sdist'
+        loadTestArgs +=" --sdk_location=${project.rootDir}/sdks/python/build/apache-beam.tar.gz"
```

Comment:
I am not  sure if there is variable for this dir, but `buildDir` points to `sdks/python/apache_beam/testing/load_tests` which after changes in sdist task in no more dist dir.

Reply:
Thank you for the fix! Sorry for missing this test when I did sdist change. There is a Gradle way to do that (instead building relative path): (e.g. [test-suites/dataflow/py35/build.gradle](https://github.com/apache/beam/blob/master/sdks/python/test-suites/dataflow/py35/build.gradle))

1. Declare a new dependency for this tarball file:
```
dependencies {
  distTarBall project(path: ":sdks:python", configuration: "distTarBall")
}
```
2. Get absolute path by `files(configurations.distTarBall.files).singleFile`.

Reply:
Thank you for help :) I changed gradle according to those hints.